internal/MozziGuts_impl_RP2040.hpp

```diff
@@ -231,11 +263,21 @@ static void startAudio() {
 #      error Audio channel pins for stereo or HIFI must be on the same PWM slice (which is the case for the pairs (0,1), (2,3), (4,5), etc. Adjust MOZZI_AUDIO_PIN_1/2 .
 #    endif
   gpio_set_function(MOZZI_AUDIO_PIN_2, GPIO_FUNC_PWM);
+  gpio_set_drive_strength(MOZZI_AUDIO_PIN_2, GPIO_DRIVE_STRENGTH_12MA); // highest we can get*/
+
+  gpio_set_drive_strength(MOZZI_AUDIO_PIN_1, GPIO_DRIVE_STRENGTH_12MA); // highest we can get
+  #  if (MOZZI_AUDIO_CHANNELS > 1)
+#    if ((MOZZI_AUDIO_PIN_1 / 2) != (MOZZI_AUDIO_PIN_1 / 2))
+#      error Audio channel pins for stereo or HIFI must be on the same PWM slice (which is the case for the pairs (0,1), (2,3), (4,5), etc. Adjust MOZZI_AUDIO_PIN_1/2 .
```

Comment:
Is this a requirement with PWMAudio, too?

Reply:
Yup! (Can't find the reference ATM, but I read that somewhere in PWMAudio)